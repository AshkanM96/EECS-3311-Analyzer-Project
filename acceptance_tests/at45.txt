open_set_enumeration
open_set_enumeration
integer_constant (1)
boolean_constant (false)
close_set_enumeration
open_set_enumeration
integer_constant (1)
boolean_constant (false)
close_set_enumeration
close_set_enumeration
evaluate
reset
reset
close_set_enumeration
