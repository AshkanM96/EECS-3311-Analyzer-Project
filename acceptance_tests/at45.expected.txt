  Expression currently specified: ?
  Report: Expression is initialized.
->open_set_enumeration
  Expression currently specified: {?}
  Report: OK.
->open_set_enumeration
  Expression currently specified: {{?}}
  Report: OK.
->integer_constant(1)
  Expression currently specified: {{1, ?}}
  Report: OK.
->boolean_constant(False)
  Expression currently specified: {{1, False, ?}}
  Report: OK.
->close_set_enumeration
  Expression currently specified: {{1, False}, ?}
  Report: OK.
->open_set_enumeration
  Expression currently specified: {{1, False}, {?}}
  Report: OK.
->integer_constant(1)
  Expression currently specified: {{1, False}, {1, ?}}
  Report: OK.
->boolean_constant(False)
  Expression currently specified: {{1, False}, {1, False, ?}}
  Report: OK.
->close_set_enumeration
  Expression currently specified: {{1, False}, {1, False}, ?}
  Report: OK.
->close_set_enumeration
  Expression currently specified: {{1, False}, {1, False}}
  Report: OK.
->evaluate
  Expression currently specified: {{1, False}, {1, False}}
  Report: Error (Expression is not type-correct).
->reset
  Expression currently specified: ?
  Report: OK.
->reset
  Expression currently specified: ?
  Report: Error (Initial expression cannot be reset).
->close_set_enumeration
  Expression currently specified: ?
  Report: Error (Set enumeration is not being specified).
